composer.json

```diff
@@ -33,6 +33,9 @@
             "GuzzleHttp\\Tests\\": "tests/"
         }
     },
+    "suggest": {
+        "psr/log": "This package holds all interfaces/classes/traits related to PSR-3 in order to use Middleware::log()"
```

Comment:
I would prefer something shorter which simply states why we suggest this package (like "Required for using the Log middleware".